index.js

```diff
@@ -1,6 +1,16 @@
+/**
+ *   understand/
+ * Facebook messenger channel purely for support channel.
+ * These messages will directly communicate with the KB service.
+ * If the KB answer not found, message forward to the owner through any one of the channels.
+ * Default support channel is Qwert channel.
+ * User can change the support channel by using command /use_for_support telegram.
+ *
+**/
+
```

Comment:
This code and commenting is very clean and superbly commented. Very well done!

Reply:
thank you @theproductiveprogrammer